docs/jwst/assign_wcs/arguments.rst

```diff
@@ -0,0 +1,30 @@
+Step Arguments
+==============
+
+The ``assign_wcs`` step has the following optional arguments to control
+the behavior of the processing.
+
+``--sip_approx`` (boolean, default=True)
+  A flag to enable the computation of a SIP approximation for
+  imaging modes.
+
+``--sip_max_pix_error`` (float, default=0.1)
+  Maximum error for the SIP forward fit, in units of pixels.
+
+``--sip_degree`` (integer, max=6, default=None)
+  Polynomial degree for the forward SIP fit. "None" uses the best fit.
+
+``--sip_max_inv_pix_error`` (float, default=0.1)
+  Maximum error for the SIP inverse fit, in units of pixels.
```

Comment:
Same as above - this is ignored if `sip_inv_degree` is specified.

Reply:
Done.